assets/js/dashboard/stats/reports/list.js

```diff
@@ -159,8 +174,19 @@ export default function ListReport(props) {
   }
 
   function renderReportHeader() {
-    const metricLabels = metrics.map((metric) => {
-      return (<span key={metric.name} className="text-right" style={{minWidth: colMinWidth}}>{ metricLabelFor(metric, props.query) }</span>)
+    let i = 0
+
+    const metricLabels = validatedMetrics().map((metric) => {
+      i++
+      return (
+        <div
+          key={metric.name}
+          className={`text-right ${hiddenOnMobileClass(i)}`}
+          style={{minWidth: colMinWidth}}
+        >
+            { metricLabelFor(metric, props.query) }
+        </div>
+      )
     })
```

Comment:
I believe the second element of the map inner function is the index. In this case you wouldn't need to declare `i` or that extra `i++`

```suggestion
    const metricLabels = validatedMetrics().map((metric, i) => {
      return (
        <div
          key={metric.name}
          className={`text-right ${hiddenOnMobileClass(i)}`}
          style={{minWidth: colMinWidth}}
        >
            { metricLabelFor(metric, props.query) }
        </div>
      )
    })
```

Reply:
Ah, cool! Good to know, but in the end I had to do it differently, because I realised we have to decide per metric, which ones to show or not. Currently, in the Goal Conversions section, on mobile view we only show two metrics - `Uniques` (`visitors`) and `CR`. And since we want to remove the `Total` (`events`) on mobile, we cannot rely on the order anymore. https://github.com/plausible/analytics/pull/3209/commits/96c11c5417afed6f478fcc2a1a148fb4b122986a

Reply:
That actually feels like a better and more explicit way of handling this problem, so I think we're good :)